### PR TITLE
misc: use time -v in time log

### PR DESCRIPTION
**Pull request by eastonman**
This results in the following output.

Screenshot 2024-05-11 at 19 22 37

### CODE REVIEW

File: verilator.mk [hunk 1]
```diff
@@ -139,7 +139,7 @@ EMU_COMPILE_FILTER =
 # 2> $(BUILD_DIR)/g++.err.log | tee $(BUILD_DIR)/g++.out.log | grep 'g++' | awk '{print "Compiling/Generating", $$NF}'
 
 build_emu_local: $(EMU_MK)
-	@echo "\n[g++] Compiling C++ files..." >> $(TIMELOG)
+	@echo -e "\n[c++] Compiling C++ files..." >> $(TIMELOG)
```
Comment (poemonsense) [inline verilator.mk:142]: what is `-e` for

Comment (eastonman) [reply verilator.mk:142]: From manual of GNU Coreutils 8.32
```
  -e     enable interpretation of backslash escapes
```

Comment (eastonman) [reply verilator.mk:142]: Otherwise it just prints like this
Screenshot 2024-05-11 at 19 42 41

Comment (poemonsense) [reply verilator.mk:142]: Look good. But I would recommend separate the changes into two PRs next time and explain each of them in git or PR comments.

**Review by poemonsense — APPROVED** [review]

**Pull request merged**